zetta_utils/common/signal_handlers.py

```diff
@@ -4,18 +4,13 @@
 
 @contextmanager
 def custom_signal_handler_ctx(fn, target_signal):  # pragma: no cover
-    # save original SIGINT handler
     original_sigint_handler = signal.getsignal(target_signal)  # eg signal.SIGINT
```

Comment:
Needs to be renamed to `original_signal_handler`.

Reply:
k